### ISSUE TITLE
Wrong information about noise profile for Canon EOS 90D on https://www.darktable.org/resources/camera-support

### ISSUE THREAD

**Issue by ricardojfc** [darktable-org/dtdocs#594]
According to [PR 10897](https://github.com/darktable-org/darktable/pull/10897), support for noise profile on Canon EOS 90D  was added on version 3.8.x, however, on the [current web site](https://www.darktable.org/resources/camera-support/), it says **no** for the `Noise profile` column on Canon EOS 90D.

**Comment by paperdigits** [comment]
Yes we have been looking for a way to auto generate these tables from the source code, because keeping them up to date by hand is cumbersome.

**Comment by elstoc** [comment]
This issue should probably be raised against the dtorg repository. This repository is for the documentation under docs.darktable.org

**Comment by kmilos** [comment]
See https://github.com/darktable-org/dtorg/issues/199 and https://github.com/darktable-org/dtorg/issues/211
